common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,9 +244,38 @@ class ModuleI18nService(object):
     i18n service.
 
     """
-
-    def __getattr__(self, name):
-        return getattr(django.utils.translation, name)
```

Comment:
I'm not entirely sure we want to remove __getattr__ here -- are there other instances of ModuleI18nService calls which would require support for a clear pass-through to django.utils.translation?  Otherwise an AttributeError would be raised and the operation added to the service, right?